package.json

```diff
@@ -27,7 +29,32 @@
     "js-sha256": "^0.9.0"
   },
   "devDependencies": {
-    "@babel/runtime": "^7.4.5",
-    "@redsift/rollup-bundler": "^0.7.1"
+    "@babel/core": "^7.9.6",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
+    "@rollup/plugin-babel": "^5.0.2",
+    "@rollup/plugin-commonjs": "^12.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.0.0",
+    "eslint-plugin-react": "^7.20.0",
+    "husky": ">=4",
+    "lint-staged": ">=10",
+    "prettier": "^2.0.5",
+    "rollup": "^2.10.9"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+  },
+  "husky": {
```

Comment:
_scope creep_: Should we add `engine` declaring build support for 12 and up?
```
"engines": {
    "node": ">=12.14.0",
    "npm": ">=6.13.4"
  },
```
See:

https://github.com/redsift/sift-oninbox-admin/blob/master/frontend/package.json#L12-L15